### DISCUSSION TITLE
Teams

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60-2024/sdy60#5] [Q&A]
Όνομα ομάδας: delta
GitHub organisation url: [delta](url)
Νέα μέλη δεκτά: ΝΑΙ/ΟΧΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2003001 Design | gh link |
| 2 | 2003002 Operations | gh link |
| 3 | ... | ... |

**Comment by epidrome** [comment]
Όνομα ομάδας: alpha
GitHub organisation url: [alpha](url)
Νέα μέλη δεκτά: ΝΑΙ/ΟΧΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2003001 | gh link |
| 2 | 2003002 | gh link |
| 3 | ... | ... |

**Reply by epidrome** [reply]
add me!

**Comment by Pagiatis-Spiros** [comment]
# ΟΜΑΔΑ 1

| α/α | Όνομα |
|---|---|
| 1 | Μαρίνα Γιαννακάκου  |
| 2 | Ανδρέας Παπαδόπουλος |
| 3 | Τάσος Παναγιωτόπουλος |
| 4 | Σπύρος Παγιάτης |

**Reply by VagNta** [reply]
ΟΜΑΔΑ 2

α/α  ΕΠΩΝΥΜΟ-ΟΝΟΜΑ 
1.     ΚΟΒΑΣ ΒΑΣΙΛΕΙΟΣ
2.     ΚΟΚΚΑΛΗΣ ΒΑΣΙΛΕΙΟΣ
3.     ΞΥΔΑΚΗΣ ΜΙΧΑΗΛ
4.     ΝΤΑΟΥΤΗΣ ΕΥΑΓΓΕΛΟΣ

**Comment by apapost** [comment]
Αποστολίδης Απόστολος (τους άλλους δεν τους ξέρω)!

**Reply by ozagoras** [reply]
Ούτε εγώ έχω ομάδα Απόστολε μπορούμε
να κάνουμε με τους υπόλοιπους
